### ISSUE TITLE
Error if running makeimage.sh

### ISSUE THREAD

**Issue by sombrowsky** [BtbN/FFmpeg-Builds#329]
I have the lastest Git-revision: 96dda7e of FFmpeg-Builds.
I am using the bash within Windows11, but I have the same error within 
an ubuntu-Virtual-Box environment.
If I use the command:
./makeimage.sh win64 gpl-shared
or
./makeimage.sh win64 gpl-shared 6.1

the folling errors occur:
 => ERROR [layer-50-vulkan  6/13] RUN --mount=src=scripts.d/50-vulkan/50-shaderc.sh,dst=/stage.sh run_stage /stag  4.0s
 => CANCELED [layer-50-librist 6/6] RUN --mount=src=scripts.d/50-librist/50-librist.sh,dst=/stage.sh run_stage /s  4.1s
 => CANCELED [layer-50-aribb24 6/6] RUN --mount=src=scripts.d/50-aribb24/50-libaribb24.sh,dst=/stage.sh --mount=s  4.1s
 => [layer-50-libjxl 4/9] ADD .cache/downloads/45-lcms2_7aaa094067389721abf0034db974d1d3db2dbcf8af5d124889b19465c  1.9s
 => CANCELED [layer-50-x265 3/3] RUN --mount=src=scripts.d/50-x265.sh,dst=/stage.sh run_stage /stage.sh            0.8s

I use FFMpeg in my Homepage with free Software for everyone:
https://www.somby.de
best regards
Reiner Sombrowsky

**Comment by BtbN** [comment]
That's not the actual error. Please scroll up and find what's actually failing.

**Comment by sombrowsky** [comment]
If I scroll up I found:
 => CANCELED [layer-50-x265 3/3] RUN --mount=src=scripts.d/50-x265.sh,dst=/stage.sh run_stage /stage.sh           14.7s

**Comment by BtbN** [comment]
Yes, all other stages got cancelled because `layer-50-vulkan 6/13` had an error.
Please find the actual error.

**Comment by sombrowsky** [comment]
Sorry, but I am using only your scripts to
make the actual versions of ffmpeg for my own
software developments for my homepage.
I cannot looking for the actual error.

**Comment by BtbN** [comment]
Then I'm unfortunately not able to help you.
The build works fine for me, and all you shared so far boils down to "There was an error".

Really, the actual error is in the log, all you need to do is scroll up a bit until you get to it.

**Comment by sombrowsky** [comment]
I have checked all with 32bit with
makeimage.sh and build.sh Script and I
cannot build it

**Comment by BtbN** [comment]
So is it win64 or win32 that's failing?
win64 builds fine for me, I didn't test win32 in a bit.

But really, without the actual error, there is nothing I can do to help you.

**Comment by sombrowsky** [comment]
Sorry if I disturb again. Today I tried to run the script again and I found the following error lines at the end: 
.....
3.088 -- Configuring incomplete, errors occurred!
3.207 CMake Error at third_party/CMakeLists.txt:80 (message):
3.207   SPIRV-Tools was not found - required for compilation
.....
ERROR: failed to solve: process "/bin/sh -c run_stage /stage.sh" did not complete successfully: exit code: 1
+ rm -f /tmp/tmp.MyCWkVYXfH.toml

After I have installed SPIRV-Tools the same error occurs.
